WordPress/WordPressTest/MockWordPressOrgXMLRPCApi.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+@objc class MockWordPressOrgXMLRPCApi: NSObject, WordPressOrgXMLRPC {
+
+    var methodPassedIn: String? = nil
+    var parametersPassedIn: [AnyObject]? = nil
```

Comment:
The main issue I see with this mock class is that it assumes that it's only going to be called once, as any `callMethod` will overwrite the methods/parameters variable. This assumes that each remote method does (at most) one API request, which is true for the Blog remote, but not in other cases (e.g. `- [PostServiceRemoteXMLRPC trashPost:success:failure:]` calls `wp.deletePost` then `wp.getPost`).

Using the existing `OHHTTPStubs` library could be more robust for this IMO.